app/services/ccms/manual_review_determiner.rb

```diff
@@ -10,29 +10,44 @@ class ManualReviewDeterminer
     attr_reader :legal_aid_application
 
     delegate :cfe_result,
+             :dwp_override,
              :passported?,
              :non_passported?,
              :has_restrictions?, to: :legal_aid_application
 
     delegate :capital_contribution_required?, to: :cfe_result
 
-    def self.call(legal_aid_application)
-      new(legal_aid_application).call
-    end
-
     def initialize(legal_aid_application)
       @legal_aid_application = legal_aid_application
       raise 'Unable to determine whether Manual review is required before means assessment' if legal_aid_application.cfe_result.nil?
     end
 
-    def call
-      needs_review?
+    def manual_review_required?
+      return true if dwp_override
+
+      manually_review_all_non_passported? || capital_contribution_required? || has_restrictions?
```

Comment:
as the list of review reasons is getting ever longer, maybe
```
def manual_review_required?
  reasons_to_review.any? true
end

def reasons_to_review
  [
    dwp_override,
    manually_review_all_non_passported?, 
    capital_contribution_required?, 
    has_restrictions?
  ]
end
```
up to you

Reply:
The only issue with that is it has to evaluate all four conditions even if dwp_override is true.  So perhaps I'll go with:

```def manual_review_required?
  dwp_override ||
    manually_review_all_non_passported? || 
    capital_contribution_required? ||
    has_restrictions?
end```

Reply:
yeah, that was the alternative 👍